test_rwo_pvc_assign_pod_node failing in HCI provider-client setup

The test cases given below are failing in HCI provider-client setup. 
tests/manage/pv_services/test_pvc_assign_pod_node.py::TestPvcAssignPodNode::test_rwo_pvc_assign_pod_node[CephBlockPool]
tests/manage/pv_services/test_pvc_assign_pod_node.py::TestPvcAssignPodNode::test_rwo_pvc_assign_pod_node[CephFileSystem]
tests/manage/pv_services/test_pvc_assign_pod_node.py::TestPvcAssignPodNode::test_rwx_pvc_assign_pod_node[CephBlockPool]
tests/manage/pv_services/test_pvc_assign_pod_node.py::TestPvcAssignPodNode::test_rwx_pvc_assign_pod_node[CephFileSystem]
```
        ocs_version = version.get_semantic_ocs_version_from_config()
        if ocs_version >= version.VERSION_4_12 and (
            config.ENV_DATA.get("platform") != constants.FUSIONAAS_PLATFORM
            or config.ENV_DATA.get("platform")
            not in constants.HCI_PROVIDER_CLIENT_PLATFORMS
        ):
>           self.verify_access_token_notin_odf_pod_logs()

tests/manage/pv_services/test_pvc_assign_pod_node.py:118: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <tests.manage.pv_services.test_pvc_assign_pod_node.TestPvcAssignPodNode object at 0x7f5e2a093520>

    def verify_access_token_notin_odf_pod_logs(self):
        """
        This function will verify logs of kube-rbac-proxy container in odf-operator-controller-manager pod
        shouldn't contain api access token
        """
        odf_operator_pod_objs = res_pod.get_all_pods(
            namespace=config.ENV_DATA["cluster_namespace"],
            selector_label="app.kubernetes.io/name",
            selector=[constants.ODF_SUBSCRIPTION],
        )
        error_msg = "Authorization: Bearer"
        pod_log = res_pod.get_pod_logs(
>           pod_name=odf_operator_pod_objs[0].name, container="kube-rbac-proxy"
        )
E       IndexError: list index out of range

tests/manage/pv_services/test_pvc_assign_pod_node.py:46: IndexError
______ TestPvcAssignPodNode.test_rwo_pvc_assign_pod_node[CephFileSystem]
```

This check "verify_access_token_notin_odf_pod_logs" is supposed to skip in the platform but the conditional statement added in the test test_rwx_pvc_assign_pod_node didn't work as expected.

In the test test_rwx_pvc_assign_pod_node , condition is needed to skip the check.

--- a/tests/manage/pv_services/test_pvc_assign_pod_node.py
+++ b/tests/manage/pv_services/test_pvc_assign_pod_node.py
@@ -110,11 +110,11 @@ class TestPvcAssignPodNode(ManageTest):
         pod.get_fio_rw_iops(pod_obj)
 
         ocs_version = version.get_semantic_ocs_version_from_config()
-        if ocs_version >= version.VERSION_4_12 and (
-            config.ENV_DATA.get("platform") != constants.FUSIONAAS_PLATFORM
-            or config.ENV_DATA.get("platform")
-            not in constants.HCI_PROVIDER_CLIENT_PLATFORMS
-        ):
+        if ocs_version >= version.VERSION_4_12 and config.ENV_DATA.get(
+            "platform"
+        ) not in constants.HCI_PROVIDER_CLIENT_PLATFORMS + [
+            constants.FUSIONAAS_PLATFORM
+        ]:
             self.verify_access_token_notin_odf_pod_logs()
 
     @acceptance
@@ -209,7 +209,9 @@ class TestPvcAssignPodNode(ManageTest):
             pod.get_fio_rw_iops(pod_obj)
 
         ocs_version = version.get_semantic_ocs_version_from_config()
-        if (ocs_version >= version.VERSION_4_12) and (
-            config.ENV_DATA.get("platform") != constants.FUSIONAAS_PLATFORM
-        ):
+        if ocs_version >= version.VERSION_4_12 and config.ENV_DATA.get(
+            "platform"
+        ) not in constants.HCI_PROVIDER_CLIENT_PLATFORMS + [
+            constants.FUSIONAAS_PLATFORM
+        ]:
             self.verify_access_token_notin_odf_pod_logs()

--- a/tests/manage/pv_services/test_pvc_assign_pod_node.py
+++ b/tests/manage/pv_services/test_pvc_assign_pod_node.py
@@ -95,7 +95,7 @@ class TestPvcAssignPodNode(ManageTest):
             config.ENV_DATA["platform"].lower()
             in constants.HCI_PROVIDER_CLIENT_PLATFORMS
         ):
-            timeout = 180
+            timeout = 300
         helpers.wait_for_resource_state(
             resource=pod_obj, state=constants.STATUS_RUNNING, timeout=timeout
         )

--- a/tests/manage/pv_services/test_pvc_assign_pod_node.py
+++ b/tests/manage/pv_services/test_pvc_assign_pod_node.py
@@ -95,7 +95,7 @@ class TestPvcAssignPodNode(ManageTest):
             config.ENV_DATA["platform"].lower()
             in constants.HCI_PROVIDER_CLIENT_PLATFORMS
         ):
-            timeout = 300
+            timeout = 180
         helpers.wait_for_resource_state(
             resource=pod_obj, state=constants.STATUS_RUNNING, timeout=timeout
         )

